src/pages/Homepage/MonthlyGoodHomepage.vue

```diff
@@ -0,0 +1,491 @@
+<template>
```

Comment:
This template is just a copy of the current LendByCategoryHomepage for now, with the kiva classic buttons